python/tests/plugins/validators/test_validators.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+from aac.plugins.validators import get_validation_definition_from_plugin_definitions
+
+
+class TestValidationModule(TestCase):
```

Comment:
I think that this class having the same name as the one in `python/tests/plugins/validators/test_validator_implementation.py` could cause problems testing. Can you confirm both tests are actually being run?

Reply:
They do. Thanks for spotting that, copy mistake. :x